docs/api.rst

```diff
@@ -87,3 +87,4 @@ Experimental
 =============
 
 .. automethod:: groclient.Experimental.get_data_points
+.. automethod:: groclient.Experimental.get_df
```

Comment:
The `get_df` function is under `groclient.GroClient`. Or am I missing something here?